modules/apps/jaggery/emm/src/emm/units/operation-bar/public/js/operation-bar.js

```diff
@@ -282,7 +172,7 @@ function runOperation (operationName) {
         }
     } else {
         invokerUtil.post(serviceEndPoint, payload,
-            successCallback, function (textStatus) {
+                         successCallback, function (textStatus) {
                 console.log(textStatus);
```

Comment:
This error callback is only useful for debugging purposes, no response seems to be going to the user at the event of an error